hawkular-dmr-client/src/main/java/org/hawkular/dmr/api/OperationBuilder.java

```diff
@@ -529,7 +530,18 @@ public T resolveExpressions() {
 
         @SuppressWarnings("unchecked")
         public T resolveExpressions(boolean resolveExpressions) {
-            baseNode.get(ModelDescriptionConstants.RESOLVE_EXPRESSIONS).set(resolveExpressions);
```

Comment:
Right.
I suppose that as a follow-up we could also call the resolve-expression command on the result of this operation if the target is EAP6

[standalone@localhost:29999 /] :resolve-expression(expression=